app/till.rb

```diff
@@ -0,0 +1,16 @@
+require 'json'
+
+class Till
+
+file = File.read('./hipstercoffee.json')
+SHOP_DETAILS = JSON.parse(file)
+
+  def initialize()
+    @shop_details = SHOP_DETAILS
+  end
+
+  def shop_details
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)